### PR TITLE
[10983][Doc][xCluster] xCluster setup docs for how to create xCluster replications setup with geo-partitioning

### PR DESCRIPTION
**Pull request by Arjun-yb**
Adding section to the xCluster setup docs for how to create xCluster replications setup with Geo-partitioning.

### CODE REVIEW

**Review by hulien22 — COMMENTED** [review]
Looks pretty good overall, thanks for this @Arjun-yb ! 

Believe you might need to rebase locally first to deal with some of Bogdan's recent doc commits

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 1]
```diff
@@ -144,10 +144,94 @@ When both universes use different certificates, you need to store the certificat
       000030a5000030008000000000004000,000030a5000030008000000000004005,dfef757c415c4b2cacc9315b8acb539a
     ```
 
-## Verifying Replication
+## Setting up Replication with Geo-partitioning
 
-You can verify replication by stopping the workload and then using the `COUNT(*)` function on the yugabyte-target to yugabyte-source match.
+Create 2 universes(source and target) with same configurations(with same regions and zones) as below
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:149]: Changed

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 2]
```diff
@@ -144,10 +144,94 @@ When both universes use different certificates, you need to store the certificat
...
+
+For Example:
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:151]: Removed

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 3]
```diff
@@ -144,10 +144,94 @@ When both universes use different certificates, you need to store the certificat
...
+* **Regions:** EU(Paris), Asia Pacific(Mumbai) and US West(Oregon)
+* **Zones:** eu-west-3a, ap-south-1a and us-west-2a
+
+```.sh
+./bin/yb-ctl --rf 3 create --placement_info "cloud1.region1.zone1,cloud2.region2.zone2"
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:156]: makes sense, added

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 4]
```diff
@@ -144,10 +144,94 @@ When both universes use different certificates, you need to store the certificat
...
+```
+
+For Example:
+```.sh
+./bin/yb-ctl --rf 3 create --placement_info "aws.us-west-2.us-west-2a,aws.ap-south-1.ap-south-1a,aws.eu-west-3.eu-west-3a"
+```
+
+Create table, table spaces and partition tables at source and target
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:164]: Changed

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 5]
```diff
@@ -144,10 +144,94 @@ When both universes use different certificates, you need to store the certificat
...
+                  FOR VALUES IN ('IN') TABLESPACE ap_ts;
+
+CREATE TABLE transactions_us
+                  PARTITION OF transactions
+                  (user_id, account_id, geo_partition, amount, created_at,
+                  PRIMARY KEY (user_id HASH, account_id, geo_partition))
+                  DEFAULT TABLESPACE us_ts;
+```
+### Steps to create Replication(Unidirectional):
+1. Collect child table UUIDs from source universe (partition tables, here transactions_eu, transactions_in and transactions_us) - these will be used while setting up replication. To collect tables' UUID, Goto `AdminUI -> Tables` option
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:212]: Changed

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 6]
```diff
@@ -144,10 +144,94 @@ When both universes use different certificates, you need to store the certificat
...
 
+  ![xCluster_with_GP](docs/static/images/explore/yb_example_table_UUID.png)
+
+2. Run replication setup command at source universe
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:216]: Changed

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 7]
```diff
@@ -144,10 +144,94 @@ When both universes use different certificates, you need to store the certificat
...
+
+    For Example:
+    ```.sh
+    ./bin/yb-admin -master_addresses 127.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 \
+    setup_universe_replication e260b8b6-e89f-4505-bb8e-b31f74aa29f3_xClusterSetup1 \
+    127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
+    000030a5000030008000000000004000,000030a5000030008000000000004005,dfef757c415c4b2cacc9315b8acb539a
+    ```
+3. Observe replication setup(`xClusterSetup1`) in Platform UI(At Replication tab in source universe and target universe)
+4. Do some DMLs(insert/update/delete) to parent table of source and observe the replication at target side(in parent table and partition tables).
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:231]: Ok, removed

File: docs/content/latest/deploy/multi-dc/async-replication.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (polarweasel) [reply docs/content/latest/deploy/multi-dc/async-replication.md]: 👍

[the remark this answers is not in the document]
Comment (polarweasel) [reply docs/content/latest/deploy/multi-dc/async-replication.md]: Also: only use `code font` for things that are actually code or UI element names. Less is more for that, or it gets annoying to read...

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 8]
```diff
@@ -180,6 +180,91 @@ When both universes use different certificates, you need to store the certificat
...
+    setup_universe_replication <producer universe UUID>_<stream_name> \
+    <producer_master_addresses> <comma_separated_table_ids>
+    ```
+
+    For Example:
+    ```.sh
+    ./bin/yb-admin -master_addresses 127.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 \
+    setup_universe_replication e260b8b6-e89f-4505-bb8e-b31f74aa29f3_xClusterSetup1 \
+    127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
+    000030a5000030008000000000004000,000030a5000030008000000000004005,dfef757c415c4b2cacc9315b8acb539a
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:264]: Updated

File: docs/content/latest/deploy/multi-dc/async-replication.md [hunk 9]
```diff
@@ -180,6 +180,91 @@ When both universes use different certificates, you need to store the certificat
...
+                  PARTITION OF transactions
+                  (user_id, account_id, geo_partition, amount, created_at,
+                  PRIMARY KEY (user_id HASH, account_id, geo_partition))
+                  DEFAULT TABLESPACE us_ts;
+```
+
+### Steps to create Replication(Unidirectional):
+1. Collect child table UUIDs from source universe (partition tables, here transactions_eu, transactions_in and transactions_us) - these will be used while setting up replication. To collect table UUIDs, go to the `Tables` section in the Admin UI (127.0.0.1:7000).
+
+  ![xCluster_with_GP](docs/static/images/explore/yb_example_table_UUID.png)
```
[the remark this answers is not in the document]
Comment (Arjun-yb) [reply docs/content/latest/deploy/multi-dc/async-replication.md:250]: makes sense, done

**Review by hulien22 — APPROVED** [review]
LGTM, thanks @Arjun-yb !

**Pull request merged**